docs/dev/deps.rst

```diff
@@ -100,36 +99,9 @@ augmented SBOM's generation. [#asbom-vulns]_
 
 The augmented SBOM is produced automatically and asynchronously as part of an
 external process that is not contained within the repository itself. The
```

Comment:
```suggestion
The augmented SBOM is produced as part of an
external process that is not contained within the repository itself. The
```

I expect producing the augmented SBOM is no longer asynchronous with the `silkbomb augment` command.